cinema/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from cinema.views import movie_list, movie_detail
+
+app_name = "cinema"
+
+urlpatterns = [
+    path("movies/", movie_list, name="moive_list"),
+    path("movies/<int:pk>/", movie_detail, name="moive_detail"),
```

Comment:
There is a typo in the URL name 'moive_detail'. It should be 'movie_detail' to match the view function and maintain consistency.